src/snowflake/cli/app/commands_registration/command_plugins_loader.py

```diff
@@ -100,6 +100,10 @@ def _load_new_plugin(
         self._loaded_command_paths[
             loaded_plugin.command_spec.full_command_path
         ] = loaded_plugin
+
+        if self._is_external_plugin(loaded_plugin):
+            log.info("Loaded external plugin: %s", plugin_name)
+
```

Comment:
Does it make sense to log all loaded plugins?
Just mark them as internal/external.

Reply:
I thought about it and I'm not sure about internal, for now we have around 10 and log them every time you use Snowflake CLI is waste of disc space for me.

Reply:
This log is added only to be sure which plugins were loaded during CLI's execution as kind of audit logs. They are helpful because user can enable/disable external plugins. It's not possible to disable builtin plugins, so IMO there is no point of logging them (especially on INFO level).